tools/lsp/wasm_main.rs

```diff
@@ -158,6 +158,7 @@ impl Drop for ReentryGuardLock {
 const IMPORT_CALLBACK_FUNCTION_SECTION: &'static str = r#"
 type ImportCallbackFunction = (url: string) => Promise<string>;
 type SendRequestFunction = (method: string, r: any) => Promise<any>;
+type HighlightInPreviewFunction = (file: string, offset: number);
```

Comment:
This works well for now. :+1: 

(In the future, we will probably want to just make this a `MessageToPreview = (command: string, data:any)` or something like that that can be used to route all messages from lsp to preview, so we don't have to create one function for each possible thing)